api/answer.go

```diff
@@ -129,7 +130,7 @@ func PostAnswerHandler(res http.ResponseWriter, req *http.Request) {
 		Question:  ans.Question,
 		Parent:    ans.Parent,
 		UserId:    user.ID,
-		Content:   ans.Content,
+		Content:   html.EscapeString(ans.Content),
```

Comment:
I've tested this but it causes problems. If a user submit an answer with a snippet of js code this escape everything. @ali-benny have implemented the XSS escaping on the client this. She uses a custom sanitizer that works with markdown and latex in order to avoid escaping code snippets. So remove this, we just close the issue